src/app.js

```
@@ -10,6 +10,38 @@
 
+  uploadProductImage(
+    file, 
+    productId, 
+    maxSize, 
+    allowedTypes, 
+    uploadUrl, 
+    callback
+  ) {
+    const url = uploadUrl + '?productId=' + productId
+    
+    const fileType = file.type
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)